AntivirusDeployment.yaml

```diff
@@ -9,6 +9,11 @@ Parameters:
     Default: 'yes'
     Type: CommaDelimitedList
     Description: Tag values of the target EC2 instances
+  AVVersion:
+    Type: String
+    Description: The version of the antivirus to deploy
+    AllowedValues:
+    - "20.2.282.0"
```

Comment:
Why setting a single AllowedValues ? Shouldn't it be a default ? 
Or is the idea to set several AllowedValues once we have more version later ?

Reply:
The latter